sidechain/src/weights.rs

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Oh you even have the weights!

Reply:
Is that good or bad? The values are extracted from the teerex weights.

Reply:
This is good. :D